test/Microsoft.Azure.Devices.Edge.Test/Module.cs

```diff
@@ -17,16 +17,18 @@ public class Module : SasManualProvisioningFixture
         const string SensorName = "tempSensor";
         const string DefaultSensorImage = "mcr.microsoft.com/azureiotedge-simulated-temperature-sensor:1.0";
 
-        [Test]
+        [TestCase(Protocol.Mqtt)]
+        [TestCase(Protocol.Amqp)]
         [Category("CentOsSafe")]
-        public async Task CertRenew()
+        public async Task CertRenew(Protocol protocol)
```

Comment:
Upstream this to 1.2 and master as well?

Reply:
master and 1.2 have the test modified to use both mqtt and amqp